src/components/Modals/BackupModal/index.tsx

```diff
@@ -56,6 +56,7 @@ const BackupModal = () => {
    * Prompt user to select a file to import and then run importMarketplace
    */
   const importSettingsFromFile = async () => {
+  // @ts-expect-error: `showOpenFilePicker not defined in type Window`
```

Comment:
This one could probably be fixed somehow...

Reply:
`window["showOpenFilePicker"]()`

Reply:
In this case, the window type should be extended, and definitely not resolved like @SunsetTechuila suggested

Reply:
Then https://www.npmjs.com/package/@types/wicg-file-system-access

Reply:
Yeah, if that package adds the type for it, we should probably add that 👍